webapp/src/components/Variants/Edition/DraggableCommands/CommandListItem.tsx

```diff
@@ -145,6 +150,15 @@ function Item({ provided, item, style, isDragging, index, onDelete, onArgsUpdate
     setJsonData(e.updated_src);
     onArgsUpdate(index, e.updated_src);
   };
+
+  const onImport = (json: object) => {
+    if (checkCommandValidity(json)) {
+      // eslint-disable-next-line dot-notation
+      setJsonData((json as any)['args']);
+      onCommandImport(index, json);
+    }
```

Comment:
here it's just the args that is uploaded. Not the whole command.
Even if the code here prevents that, it could be weird that the user could upload a fully formed command with another action name that the one he is currently uploading the data into

Reply:
execute onSave within a trycatch right after. And, if the save fail (if the data is invalid, see previous comment on adding a check on command update in the back), revert the previous value.

Reply:
The json we import should all have the same structure, so action + args.
On the other hand I find, on the contrary, that when one replaces a command at a given position, we should replace the complete command (therefore even the name of the action) and not only the args.